src/web/app/pages-instructor/instructor-courses-page/instructor-courses-page.component.ts

```diff
@@ -319,13 +319,20 @@ export class InstructorCoursesPageComponent implements OnInit {
     .subscribe(() => {
       // Wrap in a Promise to wait for all feedback sessions to be copied
       const promise: Promise<void> = new Promise<void>((resolve: () => void) => {
+        if (result.selectedFeedbackSessionList.size < 1) {
+          this.updateProgressBarHandler(1);
```

Comment:
just curious, why is it 1? Are you taking into consideration #11876 ?

Reply:
This was just an arbitrary choice, rather progress bar would be 100% filled so I thought to use 1. And I haven't seen #11876 before. Is `1` okay for this case?

Reply:
Sorry I was initially confused. If you're updating the bar to 100%, I think the value here should be 100 instead, as the service takes a percentage value